samples/ChatApp/ChatApp.Unity/Assets/Scripts/ServerShared/Hubs/IChatHub.cs

```diff
@@ -16,6 +17,10 @@ public interface IChatHub : IStreamingHub<IChatHub, IChatHubReceiver>
 
         Task SendMessageAsync(string message);
 
+        Task GenerateException(string message);
+
+        Task UnaryGenerateException(string message);
```

Comment:
Please delete it because it is not used.

Reply:
done